js/assessments/fleschReadingEaseAssessment.js

```diff
@@ -68,20 +69,20 @@ var calculateFleschReadingResult = function( fleschReadingScore, i18n ) {
 /**
  * The assessment that runs the FleschReading on the paper.
  *
- * @param {object} paper The paper to run this assessment on
- * @param {object} researcher The researcher used for the assessment
- * @param {object} i18n The i18n-object used for parsing translations
+ * @param {object} paper The paper to run this assessment on.
+ * @param {object} researcher The researcher used for the assessment.
+ * @param {object} i18n The i18n-object used for parsing translations.
  * @returns {object} an assessmentresult with the score and formatted text.
  */
 var fleschReadingEaseAssessment = function( paper, researcher, i18n ) {
 	var fleschReadingScore = researcher.getResearch( "calculateFleschReading" );
 
-	/* translators: %1$s expands to the numeric flesch reading ease score, %2$s to a link to a Yoast.com article about Flesch ease reading score,
+	/* Translators: %1$s expands to the numeric flesch reading ease score, %2$s to a link to a Yoast.com article about Flesch ease reading score,
```

Comment:
This breaks the actual translator comments in the .pot file, we need to add an exception to the `yoast/comment-starting-capital` ESLint rule.